Classes/Core/Functional/FunctionalTestCase.php

```diff
@@ -367,8 +366,16 @@ protected function setUp(): void
 
             $localConfiguration['SYS']['trustedHostsPattern'] = '.*';
             $localConfiguration['SYS']['encryptionKey'] = 'i-am-not-a-secure-encryption-key';
-            $localConfiguration['SYS']['caching']['cacheConfigurations']['extbase_object']['backend'] = NullBackend::class;
             $localConfiguration['GFX']['processor'] = 'GraphicsMagick';
+            // Set cache backends to null backend instead of database backend let us save time for creating
+            // database schema for it and reduces selects/inserts to the database for cache operations, which
+            // are generally not really needed for functional tests. Specific tests may restore this in if needed.
+            $localConfiguration['SYS']['caching']['cacheConfigurations']['extbase_object']['backend'] = 'TYPO3\\CMS\\Core\\Cache\\Backend\\NullBackend';
```

Comment:
This cache is called `extbase` since TYPO3 v10

Reply:
Setting the "extbase" cache to a nullbackend has no benefit for the core itself.
Therefore, I decided to remove setting the invalid cache to NullBackend.

Extensions needing this or having a benefit of this can and should provide the
NullBackend override by the `configurationToUseInTestInstance = []` property.

Would this be fine for you @helhum ?